docs/test/unit-test-basics.md

```diff
@@ -16,7 +16,7 @@ ms.author: "douge"
 manager: "douge"
 ---
 # Unit Test Basics
-Check that your code is working as expected by creating and running unit tests. It's called unit testing because you break down the functionality of your program into discrete testable behaviors that you can test as individual *units*. Visual Studio Test Explorer provides a flexible and efficient way to run your unit tests and view their results in Visual Studio. Visual Studio installs the Microsoft unit testing frameworks for managed and native code. Use a *unit testing framework* to create unit tests, run them, and report the results of these tests. Rerun unit tests when you make changes to test that your code is still working correctly. When you use Visual Studio Enterprise, you can run tests automatically after every build.  
+Check that your code is working as expected by creating and running unit tests. It's called unit testing because you break down the functionality of your program into discrete testable behaviors that you can test as individual *units*. Visual Studio Test Explorer provides a flexible and efficient way to run your unit tests and view their results in Visual Studio. Visual Studio installs the Microsoft unit testing frameworks for managed and native code. Use a *unit testing framework* to create unit tests, run them, and report the results of these tests. Rerun unit tests when you make changes to test that your code is still working correctly. Visual Studio Enterprise can do this automatically with [Live Unit Testing](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing-intro) which detects tests affected by your code changes and runs them in the background as you type.
```

Comment:
nit: comma before "which detects tests..."

Reply:
@rpetrusha Would it be better for Kendra to use a relative link to the topic (just live-unit-testing-intro.md) rather than an absolute link to docs.microsoft.com?

Reply:
Ah, I missed that, @gregvanl. Yes, @kendrahavens, the link should be relative.

Reply:
Oh! Thank you! Sure.